doc/architecture/trace_file_naming.adoc

```diff
@@ -44,6 +44,12 @@ Trace file contains `MotionRequest` messages.
 `su`::
 Trace file contains `StreamingUpdate` messages.
 
+`multi`::
+Trace file contains multiple types of messages for use with multi channel trace file format.
+In this case the number-of-frames field should be the largest number of frames across all channels.
```

Comment:
Because this is the most minimal change to the existing naming convention while still sticking to its spirit, and at least still halfway useful for e.g. gauging approximate content. Dito for the next two fields.

If we want to change the naming convention, then a separate PR is the way to go - personally I would love to get rid of all the protobuf versions, but a naming convention is only useful if it does not change all of the time or has twenty different variants. Especially if it is already optional, so not actually restricting anyone.